dcor/_dcor.py

```diff
@@ -1169,3 +1174,128 @@ def distance_correlation_af_inv(
             compile_mode=compile_mode,
         ),
     )
+''' 
+A Statistically and Numerically Efficient Independence Test Based on Random Projections and Distance Covariance
+
+url: https://www.frontiersin.org/articles/10.3389/fams.2021.779841/full
+'''
+
+# During implementing u_dustance_covariance_sqr I found this error---------------------------
+# AttributeError: module 'numpy' has no attribute 'bool'.
+# `np.bool` was a deprecated alias for the builtin `bool`. To avoid this error in existing code, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
+# The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
+#     https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
+np.bool = np.bool_
+
+
+def gamma_ratio(p): return np.exp(gammaln((p+1)/2) - gammaln(p/2))  # For Calculating C_p and C_q
+
+
+@njit(fastmath=True, parallel=True, cache=True)
+def dist_sum(X):  # It is nothing but the parallelized version of pdist function-----------------------
+    '''
+    Parameters
```

Comment:
I have replaced ' ' ' by " " "

Reply:
Google style docstrings, as opposed to NumPy style docstrings, use a colon to specify sections such as `Args:` or `Returns:`, instead of placing hyphens in the line below. See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for examples.